sdk/sdk-core/aar/src/main/java/mil/emp3/core/mapgridlines/coordinates/UTMCoordinate.java

```diff
@@ -0,0 +1,726 @@
+package mil.emp3.core.mapgridlines.coordinates;
```

Comment:
The UTMCorrdiante class was translated from javascript. I will find the file and add a reference to it in the comments.